fraud/bad_encoding.go

```diff
@@ -180,6 +180,7 @@ func (p *BadEncodingProof) Validate(header *header.ExtendedHeader) error {
 // OnBEFP listens to Bad Encoding Fraud Proof and stops services immediately if it is received.
 func OnBEFP(ctx context.Context, s Subscriber, stop func(context.Context) error) {
 	var err error
+	var subscription Subscription
```

Comment:
What is the reason for moving this var up here instead of initializing it below with `:=` as before?
```suggestion
	var (
		err error
		subscription Subscription
	)
```

Reply:
No needed anymore. I thought `err` will be shadowed after creating a subscription using `:=`. But double checked and It is working correctly.